generic-server/src/plugin/container.rs

```diff
@@ -268,4 +298,20 @@ mod tests {
             PluginContainerError::Unloaded
         );
     }
+    #[test]
+    fn test_unloading() {
+        // loading container from test_loading
+        let mut container = PluginContainer {
+            loaded: false,
+            collected_routes: vec![],
+            plugins: &vec![Box::new(FirstPlugin {}), Box::new(SecondPlugin {})],
+        };
+        assert!(container.load().is_ok());
+        assert!(container.routes().is_ok());
```

Comment:
```suggestion
        assert!(container.load().is_ok());
        assert!(container.routes().is_ok());
        assert_eq!(container.collected_routes.len(), 2);
```

Reply:
I don't understand the review here

Reply:
I've added `assert_eq!(container.collected_routes.len(), 2);` to ensure that the routes were indeed loaded, in the same way it is later checked that `assert_eq!(container.collected_routes.len(), 0);`.